components/arm/arm_test.go

```diff
@@ -466,7 +479,7 @@ func TestOOBArm(t *testing.T) {
 	})
 
 	t.Run("MoveToPosition works when IB", func(t *testing.T) {
-		pose = spatialmath.NewPoseFromPoint(r3.Vector{200, 200, 200})
+		pose = spatialmath.NewPoseFromPoint(r3.Vector{-800, -232.9, 62.8})
```

Comment:
this is a magic number. Instead we can:
get the end effector's pose with `EndPosition`
get the r3 point from the pose
augment the x coordinate by -20
feed that point into `NewPoseFromPoint`